sql/topology/analyzeOneway.sql

```diff
@@ -156,56 +156,56 @@ BEGIN
     tabname=sname||'.'||tname;
     vname=tname||'_vertices_pgr';
     vertname= sname||'.'||vname;
-    raise DEBUG '     --> OK';
-    EXCEPTION WHEN raise_exception THEN
+    RAISE DEBUG '     --> OK';
+    EXCEPTION WHEN RAISE_exception THEN
       RAISE NOTICE 'ERROR: something went wrong checking the table name';
       RETURN 'FAIL';
   END;
 
   BEGIN
-       raise debug 'Checking Vertices table';
+       RAISE debug 'Checking Vertices table';
        execute 'SELECT * FROM  _pgr_checkVertTab('||quote_literal(vertname) ||', ''{"id","ein","eout"}''::text[])' into naming;
        execute 'UPDATE '||_pgr_quote_ident(vertname)||' SET eout=0 ,ein=0';
-       raise DEBUG '     --> OK';
-       EXCEPTION WHEN raise_exception THEN
+       RAISE DEBUG '     --> OK';
+       EXCEPTION WHEN RAISE_exception THEN
           RAISE NOTICE 'ERROR: something went wrong checking the vertices table';
           RETURN 'FAIL';
   END;
 
 
   BEGIN
-       raise debug 'Checking column names in edge table';
+       RAISE debug 'Checking column names IN edge table';
        SELECT * into sourcename FROM _pgr_getColumnName(sname, tname,source,2);
        SELECT * into targetname FROM _pgr_getColumnName(sname, tname,target,2);
        SELECT * into owname FROM _pgr_getColumnName(sname, tname,oneway,2);
 
 
-       perform _pgr_onError( sourcename in (targetname,owname) or  targetname=owname, 2,
+       perform _pgr_onError( sourcename IN (targetname,owname) or  targetname=owname, 2,
                        '_pgr_createToplogy',  'Two columns share the same name', 'Parameter names for oneway,source and target  must be different',
                        'Column names are OK');
 
-       raise DEBUG '     --> OK';
-       EXCEPTION WHEN raise_exception THEN
+       RAISE DEBUG '     --> OK';
+       EXCEPTION WHEN RAISE_exception THEN
           RAISE NOTICE 'ERROR: something went wrong checking the column names';
           RETURN 'FAIL';
   END;
 
   BEGIN
-       raise debug 'Checking column types in edge table';
+       RAISE debug 'Checking column types IN edge table';
        SELECT * into sourcetype FROM _pgr_getColumnType(sname,tname,sourcename,1);
        SELECT * into targettype FROM _pgr_getColumnType(sname,tname,targetname,1);
 
 
-       perform _pgr_onError(sourcetype not in('integer','smallint','bigint') , 2,
+       perform _pgr_onError(sourcetypeNOTin('integer','smallint','bigint') , 2,
```

Comment:
you removed spaces that you should have not removed
sourcetype not in -> sourcetype NOT IN